src/components/Card.js

```diff
@@ -39,21 +71,28 @@ class Card {
         this._handleCardImageClick({ name: this._name, link: this._link });
       });
 
-    this._cardElement
-      .querySelector(".card__delete-button")
-      .addEventListener("click", () => this._handleDeleteButton());
+    this.likeButton.addEventListener("click", () =>
+      this._handleLikeButton(this._id)
+    );
 
-    this._cardElement
-      .querySelector(".card__like-button")
-      .addEventListener("click", () => this._handleLikeButton());
+    this.cardTrashButton.addEventListener("click", () => {
+      this._handleDeleteCard(this._id);
+    });
   }
 
   getCardElement() {
     this._cardElement = this._getTemplate();
-    this._setEventListeners();
     this._cardElement.querySelector(".card__title").textContent = this._name;
     this._cardElement.querySelector(".card__image").src = this._link;
     this._cardElement.querySelector(".card__image").alt = this._name;
+    this.likeButton = this._cardElement.querySelector(".card__like-button");
+    this.cardTrashButton = this._cardElement.querySelector(
+      ".card__delete-button"
+    );
+    this._cardLikes = this._cardElement.querySelector(".card__like-amount");
```

Comment:
Good job that you've found these variables only once here and defined them as **class fields**